src/main/java/mate/academy/spring/validation/PasswordValidation.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.spring.validation;
+
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidation implements ConstraintValidator<Password, UserRequestDto> {
+
+    @Override
+    public boolean isValid(UserRequestDto user,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        return Objects.equals(user.getPassword(), user.getRepeatPassword())
+                && user.getPassword().length() >= 4 && !user.getPassword().isBlank();
```

Comment:
order of check should be like this:
1. check isBlank
2. check a length
3. compare passwords